tests/components/gogogate2/test_cover.py

```diff
@@ -243,13 +252,14 @@ async def test_import(
 
     entity_ids = hass.states.async_entity_ids(COVER_DOMAIN)
     assert entity_ids is not None
-    assert len(entity_ids) == 2
+    assert len(entity_ids) == 3
     assert "cover.door1" in entity_ids
     assert "cover.door1_2" in entity_ids
+    assert "cover.door2" in entity_ids
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)